ext/src/Bigint.c

```diff
@@ -25,6 +25,15 @@
 #  define INT64_MIN (-INT64_MAX-1)
 #endif
 
+/* Ensure zend_long is defined for proper conversion */
+#ifndef zend_long
```

Comment:
If `zend_long` isn't defined that means it's PHP version 5.6 or less which means `lval` is a `long`. Source: https://github.com/php/php-src/blob/PHP-5.6/Zend/zend.h#L323 This should be `typedef long zend_long`.

I would consider moving this to `php_driver.h` because this is pretty useful.